Resources/Prototypes/Maps/gusstation.yml

```diff
@@ -0,0 +1,57 @@
+- type: gameMap
+  id: GusStation
+  mapName: 'GusStation'
+  mapPath: /Maps/gusstation.yml
+  minPlayers: 0
+  stations:
+    gusstation:
+      stationProto: StandardNanotrasenStation
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Aegis {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: 'TG'
```

Comment:
This can be any custom set of letters (recommended to be 2-3 letters long) and is intended, I believe, to be the constructor of the station. It is read in game something like 'NT-TG' or just 'NTTG'. Not 100% sure.

Reply:
Yeah, line 14 is the one the comment is placed on but GitHub doesn't make that obvious. 'prefixCreator' modifies the in-game display name of the station, since the final name will be something like 'NTTG Aegis xx###' something like that